### DISCUSSION TITLE
the contributions graph does not show my commits

### DISCUSSION THREAD

**Discussion by yehiaibrahim** [community/community#29321] [Actions]
I am working on a project with several other members, whenever I push a commit, it gets pushed correctly, but it is not shown on the contributions graph on my profile, what might be the reason?

**Comment by taliba570** [comment]
Hello,
Here's the following that you can do to make sure you get your progress logged:
- Make sure you are able to see the commits that you are making? 
- Double check if the account you are using is correctly configured in your local system (if not then do so)
- Try edit a file on the github.com directly and commit it

I hope this helps, if still face issue you, I'm sure other people will also have their suggestions as well

**Reply by taliba570** [reply]
BTW, I checked you profile and I can already see some contributions but they are not latest, is it the issue on a particular repo?

**Reply by yehiaibrahim** [reply]
Thanks for your reply, yes it is related to the mlpro repo.
